twake/backend/core/src/Twake/Users/Controller/Adapters/Console/Api.php

```diff
@@ -101,8 +101,7 @@ public function invite(Request $request)
         }
         else {
             $data = [
-                "firstName" => $emails[0],
-                "lastName" => "-",
+                "name" => $emails[0],
```

Comment:
Whaaat? @chamerling doing php? No way!

Reply:
yes I know, this is my first line since years. I will stop here for years